plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -104,9 +137,12 @@ bool EuroliteProFactory::DeviceAdded(
       }
       is_mk2 = true;
     } else {
-      OLA_INFO << "Connected FTDI device could be a Eurolite "
-               << "USB-DMX512-PRO MK2 but was ignored, because "
-               << ENABLE_EUROLITE_MK2_KEY << " was false.";
+      OLA_INFO << "Connected FTDI device with serial " << info.serial
+               << " could be a Eurolite USB-DMX512-PRO MK2 but was "
+               << "ignored, because "
+               << ENABLE_EUROLITE_MK2_KEY << " was false and "
+               << "its serial number was not specifically specified with "
+               << EUROLITE_MK2_SERIAL_KEY;
       return false;
     }
```

Comment:
We need to fix the code a few lines below here, where it then ignores our special serial number and uses the device and bus number, for the MK2 at least, as that should mean the patch persists even when you plug it into a different USB port, which it won't do currently.

You should be able to see that either way as the serial number should show in the device name/description in OLA.

Is it true that ALL MK2's have a serial number (as they're using an FTDI chip?)?

Reply:
I'm not sure what you mean, can you propose the fix?

Reply:
BTW I suspect all MK2's have a different serial number indeed, they indeed use an FTDI chip.

Reply:
I'll do another PR for your to test. It just improves what this has, so this can be merged as is.

Reply:
Opened #1932 for this so will track it there.